projects/shared/src/lib/component/documents/files/files.component.html

```diff
@@ -25,20 +25,28 @@
       [responsiveOptions]="responsiveOptions"
     >
       <ng-template let-file pTemplate="item">
+        @if (file.thumbnail){
             <img
               src="{{ file.thumbnail }}"
               [alt]="file.label"
               class="w-6 shadow-1 border-300 border-1 mb-1"
             />
+          } @else {
+            <div
+            src="{{ file.thumbnail }}"
+            [alt]="file.label"
+            style="width: 80px; height: 100px; display: inline-flex; align-items: self-end;"
```

Comment:
Create a css class into the file.component.scss and make this in class attribute ?